From 7538a3c112dec087d457992bffbaad33e2009cc7 Mon Sep 17 00:00:00 2001
From: Nicolas Fella <nicolas.fella@gmx.de>
Date: Fri, 31 Jul 2020 12:38:11 +0200
Subject: [PATCH 18/21] Fix QML errors

QML is complaining about non-existant property "property" and I don't see why such a line would be needed in the first place
---
 components/qml/StartupFeedback.qml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/components/qml/StartupFeedback.qml b/components/qml/StartupFeedback.qml
index 2300bab..0e0481e 100644
--- a/components/qml/StartupFeedback.qml
+++ b/components/qml/StartupFeedback.qml
@@ -153,14 +153,12 @@ Window {
                         }
                         XAnimator {
                             target: background
-                            property: "x"
                             to: 0
                             duration: units.longDuration
                             easing.type: Easing.InOutQuad
                         }
                         YAnimator {
                             target: background
-                            property: "y"
                             to: 0
                             duration: units.longDuration
                             easing.type: Easing.InOutQuad
-- 
2.26.2

